sonic-net/sonic-buildimage
# PDE Migration to bookworm in sonic-buildimage

#### Why I did it
Upgrade docker-pde to bookworm

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Update the relevant Makefile and Dockerfile to have use bookworm version of debian

#### How to verify it
Login to pde docker to see the version of debian updated to bookworm in /etc/os-release output



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
